api/types/src/move_types.rs

```diff
@@ -38,21 +38,69 @@ impl From<AnnotatedMoveStruct> for MoveResource {
 #[derive(Clone, Debug, PartialEq)]
 pub struct U64(u64);
 
+impl U64 {
+    pub fn into_inner(&self) -> u64 {
+        self.0
+    }
+}
+
+impl From<u64> for U64 {
+    fn from(d: u64) -> Self {
+        Self(d)
+    }
+}
```

Comment:
Want to add a From impl for the other way too?

`impl From<U64> for u64 { ... }`